clientv3/txn_test.go

```diff
@@ -98,4 +100,6 @@ func TestTxnPanics(t *testing.T) {
 			t.Errorf("#%d: did not panic, wanted panic %s", i, tt.err)
 		}
 	}
+
+	testutil.WaitPanic()
```

Comment:
There's some extra runtime processing when handling a defer in a panic. Since there's no way to join a goroutine, the test fails as if it were doing something like:

```go
defer func() {
    errc <- recover()
    time.Sleep(10 * time.Second)
}()
panic("kaboom")
```
(the test fails as a leaked goroutine in a runtime loop at https://golang.org/src/runtime/panic.go#L413)

Another possible (probably better) way to handle this is to filter out runtime/panic in `interestingGoRoutines`. What do you think?